### ISSUE TITLE
Support Runtime-typed Unflattening

### ISSUE THREAD

**Issue by SteveWilkes** [agileobjects/AgileMapper#87]
Re: #79, currently can do this:

```C#
Mapper.Unflatten(queryString).To<TResult>();
```

...should also be able to do this:

```C#
Mapper.Unflatten(queryString).To(resultType);
```

**Comment by SteveWilkes** [comment]
Added to the latest code, release to follow.

**Comment by YandyZaldivar** [comment]
I noticed in your post you are referring to `ToANew` while in the code it is `To`.

**Comment by SteveWilkes** [comment]
So it is! Fixed. Thanks :)

**Comment by SteveWilkes** [comment]
This is included in [v0.27](https://www.nuget.org/packages/AgileObjects.AgileMapper).

Cheers! :)
